hasjob/search.py

```diff
@@ -1,83 +1,45 @@
 # -*- coding: utf-8 -*-
 
-import os.path
 from flask.ext.sqlalchemy import models_committed
 from flask.ext.rq import job
-import time
-from sqlalchemy.exc import ProgrammingError
-from whoosh import fields, index
-from whoosh.index import LockError
-from whoosh.qparser import QueryParser
-from whoosh.analysis import StemmingAnalyzer
-
 from hasjob import models, app
 
+from pyelasticsearch import ElasticSearch
+es = ElasticSearch('http://localhost:9200/')
```

Comment:
Move this to `hasjob/__init__.py` and don't hardcode the URL here. Load from config in `init_for`.